scripts/processing/06-make_epochs.py

```diff
@@ -77,42 +80,50 @@ def run_epochs(subject_id, tsss=False):
 
         raw = mne.io.read_raw_fif(run_fname)
 
-        delay = int(round(0.0345 * raw_list[-1].info['sfreq']))
+        delay = int(round(0.0345 * raw.info['sfreq']))
         events = mne.read_events(op.join(data_path, 'run_%02d-eve.fif' % run))
         events[:, 0] = events[:, 0] + delay
         events_list.append(events)
 
         raw.info['bads'] = all_bads
         raw_list.append(raw)
     raw, events = mne.concatenate_raws(raw_list, events_list=events_list)
-    raw.set_eeg_reference()
+    raw.set_eeg_reference(projection=True)
 
     picks = mne.pick_types(raw.info, meg=True, eeg=True, stim=True,
-                           eog=True)
+                           eog=True, exclude=())
 
     # Epoch the data
+    print('  Epoching')
     epochs = mne.Epochs(raw, events, events_id, tmin, tmax, proj=True,
                         picks=picks, baseline=baseline, preload=True,
                         decim=5, reject=None)
+    print('  Interpolating bad channels')
     epochs.interpolate_bads()
+    print('  Getting rejection thresholds')
     reject = get_rejection_threshold(epochs)
     epochs.drop_bad(reject=reject)
+    print('  Dropped %0.1f%% of epochs'
+          % (100 * np.mean([len(x) > 0 for x in epochs.drop_log])))
```

Comment:
isn't there a function in MNE which gives the same info? just want to keep the code simple as it's meant to be educational as well

Reply:
not that I know of. `epochs.selection` gives indices into the original but that won't tell you how many there originally were

Reply:
what about `epochs.drop_log_stats()` ?